### ISSUE TITLE
REcaptcha not working

### ISSUE THREAD

**Issue by degiorgig** [JicuNull/CoinMarketCapLoyalty#2]
Hi I was able to run it but the service is not working

{
    "status": {
        "timestamp": "2022-02-19T07:57:06.650Z",
        "error_code": "500",
        "error_message": "Recaptcha invalid",
        "elapsed": "1",
        "credit_count": 0
    }
}
any idea?

**Comment by Louuke** [comment]
Yes, CoinMarketCap changed the captcha to a slider captcha some time ago, which is very similar to a GeeTest captcha. Since then, the code is broken.

You could solve the captcha by using Selenium and calculate how far it needs to move the captcha slider.

**Comment by degiorgig** [comment]
thanks, can't we just use the geetest captcha?
